File: models/python/person_model.py
class Person:
   def __init__(self, firstName, lastName):
      self.firstName = firstName
      self.lastName = lastName

   def getFullName(self):
      return self.firstName + ' ' +  self.lastName

   def getInitial(self):
      return self.firstName[0] + '.' + self.lastName[0]

mike = Person("Michael", "Johnson")
print(mike.getFullName())
print(mike.getInitial())

carly = Person("Carly", "Angelo")
print(carly.getFullName())
print(carly.getInitial())

jessie = Person("Jessie", "Raelynn")
print(jessie.getFullName())
print(jessie.getInitial())

File: models/java/person_model.java
class Person {
   // ここからクラス記述してください
      public String firstName;
      public String lastName;

      public Person(String firstName, String lastName){
         this.firstName = firstName;
         this.lastName = lastName;
      }

      public String getFullName(){
         return this.firstName + " " + this.lastName;
      }

      public String getInitial(){
         return this.firstName.charAt(0) + "." + this.lastName.charAt(0);
      }
}

class MyClass{
   public static void main(String[] args){
      Person mike = new Person("Michael", "Johnson");
      System.out.println(mike.getFullName());
      System.out.println(mike.getInitial());

      Person carly = new Person("Carly", "Angelo");
      System.out.println(carly.getFullName());
      System.out.println(carly.getInitial());

      Person jessie = new Person("Jessie", "Raelynn");
      System.out.println(jessie.getFullName());
      System.out.println(jessie.getInitial());
   }
}
